lib/cisco_node_utils/interface_hsrp_group.rb

```diff
@@ -36,6 +36,7 @@ def initialize(interface, group_id, ip_type, instantiate=true)
 
```

Comment:
Not sure I understand the following line.
https://github.com/cisco/cisco-network-node-utils/blob/3a8e8c5d4339e5faf5ab744cc413d23f3b59f3bc/lib/cisco_node_utils/interface_hsrp_group.rb#L104

If str is nil, then it found no authentication configs.  How did you determine what the default values were for authentication in that case?  Is that even possible?